subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -60,17 +60,13 @@ static bool hawkbit_initialized;
 
 #ifndef CONFIG_HAWKBIT_DDI_NO_SECURITY
 
-#ifdef CONFIG_HAWKBIT_DDI_GATEWAY_SECURITY
-#define AUTH_HEADER_START "Authorization: GatewayToken "
-#else
-#define AUTH_HEADER_START "Authorization: TargetToken "
-#endif /* CONFIG_HAWKBIT_DDI_GATEWAY_SECURITY */
+#define AUTH_HEADER_START                                                                          \
```

Comment:
We can probably use some Kconfig magic here instead:

```kconfig
config HAWKBIT_DDI_SECURITY_HEADER_START
    string
    default "Authorization: GatewayToken " if HAWKBIT_DDI_GATEWAY_SECURITY
    default "Authorization: TargetToken " if HAWKBIT_DDI_TARGET_SECURITY
    help
      Internal helper Kconfig blah blah blah
```

Reply:
For the first we will leave it like it is.